### ISSUE TITLE
Dragon API Invocation Target Exception

### ISSUE THREAD

**Issue by LighttronPrime** [ReikaKalseki/Reika_Mods_Issues#2248]
Today I updated DragonAPI, Reactorcraft, and Rotarycraft in my large scale modpack. The pack now crashes when loading every time. This crash does not produce a crash report, only a generic "an unexpected error has occurred" message on the MC launcher. I tried loading the mods in a small test pack, and they loaded without issue. I assume this crash is occurring due to a conflict with one of the many other mods in my pack, but only DragonAPI is referenced in the log, and then only indirectly. This is the only change I made to my modlist before loading the pack. Any help would be appreciated, but for the time being I'll move back to version 23b.

[latest.log](https://github.com/ReikaKalseki/Reika_Mods_Issues/files/3996790/latest.log)

**Comment by ReikaKalseki** [comment]
You have some other mod ASMing into the renderer. Which is it?

**Comment by ReikaKalseki** [comment]
Someone mentioned a compatibility issue with BetweenLands; is that in your pack?

**Comment by LighttronPrime** [comment]
The betweenlands is in my modpack.

I tried removing optifine, which tends to be the source of these kinds of issues, and got an actual crash report (I should have tried this immediately in retrospect). By this point, my modlist has moved around a little bit, but the crash still happens under the same conditions: everything loads fine with v23b, and crashes with v24c. I can't make heads or tales of this crash report, and it doesn't specifically mention any of your mods outside of the long list of all the mods installed. Hopefully this helps.

[crash-2020-01-05_21.48.49-client.txt](https://github.com/ReikaKalseki/Reika_Mods_Issues/files/4024537/crash-2020-01-05_21.48.49-client.txt)

**Comment by ReikaKalseki** [comment]
See https://github.com/Angry-Pixel/The-Betweenlands/issues/950

**Comment by LighttronPrime** [comment]
Yes, Betweenlands was the problem, and the JVM argument fixed my issue.  Thanks for the assistance.
